packages/blast-utilities/fs-utils.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs')
+const { getProjectRootPath } = require('./package-info')
+const BlastError = require('./blast-error')
+
+function saveAccounts(accounts) {
+  const pkgRoot = getProjectRootPath()
```

Comment:
I think it is more accurate to name the variable after "project root" as we have `getProjectRootPath() `for initialized root and `getPackageRootPath()` for CLI package root